src/main/java/com/microsoft/azure/vmagent/AzureVMAgentTemplate.java

```diff
@@ -217,6 +223,9 @@ public AzureVMAgentTemplate(
     }
 
     public String isType(final String type) {
```

Comment:
it's used in the config.jelly file to determine if a radio button is checked or not. For some reason it needs a string for the boolean